### DISCUSSION TITLE
Hello everyone, Kashan here.

### DISCUSSION THREAD

**Discussion by kashan2a** [OpenMined/courses#20] [Introduce Yourself]
Hello everyone I am Software Developer and Data Science Graduate Student right now, looking forward to meeting you all.

**Comment by yemikifouly** [comment]
Hello Kashan! 🙂 Welcome! Are you doing a Master or a PhD? We have a lot of graduate students in the community and from so many different fields too so it is always interesting to hear about what a new member is studying.

**Reply by kashan2a** [reply]
I am doing a Masters in Data Science, interested in the field of NLP.
